src/helpers/hotkeysManager.js

```diff
@@ -65,7 +65,11 @@ WebViewer(...)
     }
 
     // https://github.com/jaywcjlove/hotkeys#defining-shortcuts
-    hotkeys(key, handler);
+    if (key === 'space') {
+      hotkeys(key, { keyup: true }, handler);
+    } else {
+      hotkeys(key, handler);
```

Comment:
One thing I forgot to mention is that users can define the UI hotkeys in their app by calling
```javascript
// see hotkeys.js
instance.hotkeys.on('space', () => {});
instance.hotkeys.off('space');
```

If we only have `keyup: true` for this key then when they define hotkeys they will see that their handler for `space` is triggered twice and this may confuse them. I'm not sure if I'm making much sense, we can discuss more about this.

I think it's actually useful to have `{ keyup: true }` here since users may want to do something on `keyup`. What do you think about this? If we have `hotkeys(key, { keyup: true }, handler);` for every key then we may need to tweak the existing logic a bit.